### DISCUSSION TITLE
UI Framework like Chakra/Mantine?

### DISCUSSION THREAD

**Discussion by mmailaender** [sveltejs/kit#7984] [Q&A] [answered]
After playing arround with Svelte and Svelte-kit I've felt really in love. From DX it's easy to adapt and fast to develop. So some good reason to go with Svelte. 
Except the UI bothers me. Coming from React/Nextjs I was using there some polished and good documented UI frameworks to fast develop the UI (Mantine, Chakra, MaterialUI). If I'm looking into the Svelte eco context I'm not seeing any UI frameworks that are on that Level. There are some MaterialUI/tailwind clones, but they seem to have a poor documentation. 

So I'm not sure if I missing some piece of puzzle here - But is there some UI Framework existing for Svelte that is on the same level like Chakra, Mantine, MaterialUI (From documentation quality and component richness)? Or is there another way to go in Svelte?

**Comment by benmccann** [comment]
See https://sveltesociety.dev/components#design-systems

**Reply by mmailaender** [reply]
Yes, I know this page and the referenced UI Frameworks. So am I understanding your answer correctly as a synonym for
"_Yes, in Svelte UI Frameworks are still relevant. And no, there is currently no UI Framework existing, that is already on the level like Chakra, Mantine from Documentation quality and component richness_" ?

**Reply by benmccann** [reply]
Well the one you ended up choosing (https://github.com/sveltejs/kit/discussions/7984#discussioncomment-5067416) was the first one on the list I linked :laughing:

**Reply by mmailaender** [reply]
A lot amazing things happened in the last 8 months. 🙂 Including that Skeleton was listed on Svelte society and launched their v2. Unfortunatly it was not yet listed 8 months back because it was quite new launched.
Nevertheless, now it's listed there and as easier to find as ever 🙂

**Comment by Devr-pro** [comment]
You might find [https://daisyui.com/](https://daisyui.com/) helpful, its tailwind based UI library and works for almost every environment.

**Comment by arjarn** [comment]
Like https://flowbite-svelte.com/ : tailwind based UI library too. So it combine utilities classes and ready-to-use components.

**Comment by mmailaender** [comment, marked as answer]
I finally found one: [Skeleton](https://github.com/skeletonlabs/skeleton) has quite good documentation and is also easy to use. It provides UI components and integrates natively with Tailwind CSS for styling.

**Reply by KiddoV** [reply]
Skeleton is nice. But it is not even close to Mantine for Charka UI.

**Reply by mmailaender** [reply]
It serves two different purposes.

Skeleton is mainly a design system built up on Tailwind.
It integrates nicely with different headless ui component libraries like Bits ui, Radix or Melt Ui. They also provide their own components built on zag js, but they are definitely not yet that feature rich as Mantine or Chakra from component side.

Mantine and Chakra on the other hand are mainly component libraries and are quite limited on design system side.

So if you're missing components in Skeleton, like I did, I recommend going with e.g. [Bits UI and use Skeleton as a design system on top](https://www.skeleton.dev/docs/headless/bits-ui).

**Reply by KiddoV** [reply]
Because you are asking for something mostly similar to MantineUI, maybe this should be a better choice: https://svelteui.dev/ - It is in `beta`, but I feel it is similar to Mantine because of the docs layout and component richness.

They even state in the docs:
```txt
SvelteUI is a Svelte library (heavily inspired by but not affiliated with [Mantine](https://mantine.dev/))...
```
So, if I love Mantine, I would go with this lib.

**Reply by mmailaender** [reply]
As this thread is already two years old, I've settled already with Skeleton + Bits UI and are pretty happy with it - thanks 🙂
